ports/renesas-ra/lwip_inc/lwipopts.h

```diff
@@ -42,12 +42,14 @@
 extern uint32_t rng_read(void);
 #define LWIP_RAND() rng_read()
 
-// lwip takes 26700 bytes
-#define MEM_SIZE (8000)
-#define TCP_MSS (800)
-#define TCP_WND (8 * TCP_MSS)
-#define TCP_SND_BUF (8 * TCP_MSS)
-#define MEMP_NUM_TCP_SEG (32)
+#define MEM_SIZE                        (64 * 1024)
```

Comment:
This configuration will use a lot of RAM (did you measure how much?).  And I don't think it gains that much performance by using so much RAM.

How about instead use (taken from stm32 config):
```c
// lwip takes 45600 bytes
#define MEM_SIZE (16000)
#define TCP_MSS (1460)
#define TCP_WND (8 * TCP_MSS)
#define TCP_SND_BUF (8 * TCP_MSS)
#define MEMP_NUM_TCP_SEG (32)
```

Reply:
> how can I measure it ?

Just look at the amount of data/bss that is used by the firmware.elf file, using `arm-none-eabi-size`.

Reply:
@dpgeorge I tested both configs and you were right I don't see any significant difference, so I lowered mem_size to 16K (other options are the same in both configs).

Reply:
Very good!